lms/djangoapps/support/static/support/jsx/entitlements/data/actions/actionCreators.js

```diff
@@ -0,0 +1,40 @@
+import { getEntitlements } from '../api/client';
+
+export const entitlementActions = {
```

Comment:
You could also consider putting these in their own constants file.

Reply:
this doesn't feel too cluttered yet, I probably will when I add the update/create sets of actions.